src/jobs/lint-app.yml

```diff
@@ -1,9 +1,91 @@
 description: >
   This job will lint the application based on what language the application is written in.
 
-executor: node/default
+parameters:
+  # Not Required Parameters
+  docker-image:
+    type: string
+    default: "cimg/base:stable"
+    description: |
+      The docker image that will be used as an executor
+      if isNode is true, node/default is recommended
+      if isGolang is true, golangci/golangci-lint:latest-alpine is recommended
+      if isPython is true, cimg/python:3.10 is recommended
+  isNode:
+    type: boolean
+    default: false
+    description: |
+      Set `true` if wanting to lint a node.js application
+      Will use the script `npm run lint`
```

Comment:
Just as a chore when we implement the orb we will need to standardize linting on all of our node apps

Reply:
^

Reply:
We can do this as we update the config.yml